src/components/map/DriveByTimes.js

```diff
@@ -171,9 +168,6 @@ class DriveByTimes extends React.Component {
 
                 return (
                   <TimeTag
-                    onClick={onTimeClick(
```

Comment:
This component is not used at the moment and this part was causing eslint to fail.